ote/src/clj/ote/tools/git.clj

```diff
@@ -0,0 +1,7 @@
+(ns ote.tools.git
+  "Tools for Git in build process"
+  (:require [clojure.java.shell :as sh]
+            [clojure.string :as str]))
+
+(defmacro current-revision-sha []
```

Comment:
We don't win anything by using git revisions but add complexity.
I know it won't hurt us but still take into consideration that we can use something simpler, e.g. create hash from timestamp.

Reply:
git revision uniquely identifies the current version and is imo much better than a timestamp. It is also simpler because the git revision does not change during the build (like a timestamp does)

Reply:
webpack style hashing is also viable: https://github.com/alexindigo/webpack-chunk-hash